documentation/slate/source/includes/apis.md

```diff
@@ -1936,8 +1937,12 @@ For example:
 
 -   overall public transport traffic report on Ile de France coverage
     -   <https://api.navitia.io/v1/coverage/fr-idf/traffic_reports>
+-   overall public transport traffic report on Ile de France coverage with disruptions having tags passed in parameter values
+    -   <https://api.navitia.io/v1/coverage/fr-idf/traffic_reports?tags[]=bob&tags[]=bobette>
```

Comment:
Bob and Bobette in the official doc??? Seems more professional to give actual possible tags.